## Chapter 7: Bayes' Theorm with LEGO

<b>Q1. Kansas City, despite its name, sits on the border of two US states: Missouri and Kansas. The Kansas City metropolitan area consists of 15 counties, 9 in Missouri and 6 in Kansas. The entire state of Kansas has 105 counties and Missouri has 114. Use Bayes’ theorem to calculate the probability that a relative who just moved to a county in the Kansas City metropolitan area also lives in a county in Kansas. Make sure to show P(Kansas) (assuming your relative lives either in Kansas or Missouri), P(Kansas City metropolitan area), and P(Kansas City metropolitan area | Kansas).

<u><b><font color = green>Counties in Kansas City Metro<b><u>
    
    9 = Missouri
    6 = Kansas
    105 = Kansas Total
    114 = Missouri Total

<u><b><font color = green>105 = Kansas Total
    
   114 = Missouri Total

<u><b><font color = green>P(Kansas | Kansas City) =
    (P(Kansas City | Kansas) x P(Kansas)) / P(Kansas City)

In [2]:
#P(Kansas City | Kansas) = 6/105
6/105

[1] 0.05714286

In [3]:
#P(Kansas) = 105/219
15/219

[1] 0.06849315

In [4]:
#P(Kansas | Kansas City)
(6/105)*(105/219)/()

[1] 0.4

<b>Q2. A deck of cards has 52 cards with suits that are either red or black. There are four aces in a deck of cards: two red and two black. You remove a red ace from the deck and shuffle the cards. Your friend pulls a black card. What is the probability that it is an ace?

cards = 52

colors = 2

suits = 4

aces = 4

Subtract Red Ace, 

Friend pulls black card:

black cards = 26

total cards remain = 51

<b><font color = green>P(ace | black card) = (P(black card | ace)*P(ace)) / P(black card)

In [7]:
#If we pulled an Ace, the probability of it being black is
#P(black card | ace) = 2/3
2/3

[1] 0.6666667

In [8]:
#3 Aces remain in a 51 card deck
#P(ace) = 3/51
3/51

[1] 0.05882353

In [9]:
#26 out of 51 cards are black
#P(black card) = 26/51
26/51

[1] 0.5098039

In [10]:
#P(ace | black card) = (2/3)*(3/51)/(26/51)

In [12]:
(2/3)*(3/51)/(26/51)

[1] 0.07692308

In [23]:
#Convert Decimal to Fraction
ratio<-function(decimal){
  denominator=1
  while(nchar(decimal*denominator)!=nchar(round(decimal*denominator))){
    denominator=denominator+1
  }
  str=paste(decimal*denominator,"/",sep="")
  str=paste(str,denominator,sep="")
  return(str)
}

In [24]:
ratio(0.0769230769230769)

[1] "1/13"

<i><b><font color = green>The probability of the black card being an ace is 1/3 or about 7.7%